openedx_tagging/core/tagging/rest_api/v1/serializers.py

```diff
@@ -192,3 +192,67 @@ def get_children_count(self, obj):
         Returns the number of child tags of the given tag.
         """
         return len(obj.sub_tags)
+
+
+class TaxonomyTagCreateBodySerializer(serializers.Serializer):  # pylint: disable=abstract-method
+    """
+    Serializer of the body for the Taxonomy Tags CREATE view
+    """
+
+    tag = serializers.CharField(required=True)
+    parent_tag_value = serializers.CharField(
+        source='parent.value', required=False
```

Comment:
Is this `source` actually doing anything?

Reply:
I removed it and it doesn't seem like it had any impact. I think this is used more for serializing model/response data rather than handing request data.